distributed/bokeh/worker_monitor.py

```diff
@@ -59,6 +67,11 @@ def resource_append(lists, msg):
     for k in ['cpu', 'memory-percent']:
         lists[k].append(mean(pluck(k, L)) / 100)
 
+    try:
+        net = mean(pluck('network-send', L)) if list(pluck(k, L)) else 0
+    except KeyError:
+        net = 0
+    lists['network-send'].append(net / 2**20 / 0.5)
```

Comment:
This is where the value of the update interval is assumed, because the scale is a rate, not a ratio as the other lines.